aggsender/types/interfaces.go

```diff
@@ -187,7 +187,7 @@ type Logger interface {
 
 // CertificateStatusChecker is an interface defining functions that a CertificateStatusChecker should implement
 type CertificateStatusChecker interface {
-	CheckPendingCertificatesStatus(ctx context.Context) CertStatus
+	CheckPeriodicallyCertificateStatus(ctx context.Context) (CertStatus, error)
```

Comment:
I would simplify the naming and call it simply like this (as the function isn't aware whether it is called periodically or not, it simply checks the certificate status). WDYT?
```suggestion
	CheckCertificateStatus(ctx context.Context) (CertStatus, error)
```

Reply:
There are another function called `CheckInitialStatus` so it make sense that this one point that is regularly called instead of called at startup... maybe it can be rename to `CheckPeriodicallyStatus` to be more symmetrical if you think that fit better. WDYT?

Reply:
I'm fine with both approaches, maybe `CheckPeriodicallyStatus` is better aligned, but as you wish...

Reply:
done 4b5c0a8eb3f672bf7631d46f1dac58bbb9ae205a